# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/TextureClient.h
# Commit: 6a3a092a59b6
# Full Hash: 6a3a092a59b6939ff727c3b1d058cc3a857581a9
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2022-12-10 21:36:51
# Regressor Bug: 1795768
# File Overlap Count: 1
# Description:
#   Bug 1795768 - Do not use Direct2D canvas if willReadFrequently is set. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164364
# ==============================================================================

diff -r 169fe70ad296 -r 6a3a092a59b6 gfx/layers/client/TextureClient.h
--- a/gfx/layers/client/TextureClient.h	Sat Dec 10 04:46:28 2022 +0000
+++ b/gfx/layers/client/TextureClient.h	Sat Dec 10 04:46:28 2022 +0000
@@ -95,6 +95,9 @@
   // The texture is going to be updated using UpdateFromSurface and needs to
   // support that call.
   ALLOC_UPDATE_FROM_SURFACE = 1 << 7,
+
+  // Do not use an accelerated texture type.
+  ALLOC_DO_NOT_ACCELERATE = 1 << 8,
 };
 
 enum class BackendSelector { Content, Canvas };
